backend/kafka/create_topic.py

```
@@ -1,38 +1,177 @@
+
+                admin_client.create_topics(
+                    new_topics=[topic],
+                    validate_only=False
+                )
+
+                print(f"[INFO] Topic '{topic_name}' created successfully")
+
+            except KafkaError as e:
+                print(f"[ERROR] Error creating topic '{topic_name}': {e}")
+
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)